### PR TITLE
ISLANDORA-1485 Display TN in department object view.

### PR DESCRIPTION
**Pull request by rosiel**

### CODE REVIEW

**Comment by nigelgbanks** [comment]
Looks good to me.

File: theme/theme.inc [hunk 1]
```diff
@@ -148,6 +148,11 @@ function islandora_entities_get_dept_members($dept, $pid) {
  */
 function islandora_entities_preprocess_islandora_dept(array &$variables) {
   module_load_include('inc', 'islandora', 'includes/metadata');
+  $module_path = drupal_get_path('module', 'islandora');
+  drupal_add_css("$module_path/css/islandora.objects.css");
   $object = $variables['object'];
   $variables['metadata'] = islandora_retrieve_metadata_markup($object, TRUE);
+  if ($object["TN"]) {
```
Comment (DiegoPino) [inline theme/theme.inc]: Minor, not even need to modify, but for future consistency we should keep one type of quotes in the same module, double or single are both OK.

**Comment by rosiel** [comment]
Good catch, thanks @DiegoPino! Fixed in 2 places.

**Comment by DiegoPino** [comment]
Np @rosiel, thanks to you, will merge once travis finishes.

**Pull request merged**